transcript/src/main/java/com/transcriptanalyzer/transcript/User_Requests_Intents/Service/TranscriptService.java

```diff
@@ -23,21 +24,33 @@ public class TranscriptService {
 
     private ApiRepository apiRepository;
 
-//    //Stores the APIKey in the apiAccess.properties files
-//    //Switched to static
-//    public static void storeAPIKey(String apiKey) {
-//    }
-//
-//
-//    //Stores the APIVersion in the apiAccess.properties files
-//    //Switched to static
-//    public static void storeAPIVersion(String apiVersion) {
-//
-//    }
+    public static ArrayList<String> getIntents() throws IOException {
+        ArrayList<String> list = new ArrayList<>();
+    //Intent 1
+        String intent_1 = getJSONContent().get(0).get(1).get(0).replaceAll("[:{\"}]","");
+        intent_1 = intent_1.substring(11);
+        String intent_2 = getJSONContent().get(0).get(3).get(0).replaceAll("[:{\"}]","");
+        intent_2 = intent_2.substring(11);
+        String intent_3 = getJSONContent().get(2).get(1).get(0).replaceAll("[:{\"}]","");
+        intent_3 = intent_3.substring(11);
+
+
+        list.add(intent_1);
+        list.add(intent_2);
+        list.add(intent_3);
 
-    public void storeAPIInfo(API api) {
+//      System.out.println(list);
+
+        return list;
```

Comment:
I am note sure I understand which list this is returning.

Reply:
AFAIK in that iteration of the code, that was just a list of 3 handpicked intents from the Voiceflow API used for testing.